src/select/src/SelectField.js

```diff
@@ -55,6 +56,7 @@ const SelectField = memo(function SelectField(props) {
         required={required}
         isInvalid={isInvalid}
         appearance={appearance}
+        aria-describedby={generateAriaDescribedBy(id, { description, hint, validationMessage })}
```

Comment:
This `aria-describedby` isn't reaching the underlying `select` element. Instead, it's being placed on that element's wrapper.

I'd recommend changing the `Select` component to always pass `aria-describedby` down to the inner `select` element.